typescript/generate_node_modules.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+#
+# This file is distributed under the MIT License. See LICENSE.md for details.
+#
+
+set -euo pipefail
+
+SCRIPT_DIR="$( dirname -- "${BASH_SOURCE[0]}" )"
+
+cp "$SCRIPT_DIR/package.json" .
+yarn install --yarnrc "$YARNRC"
```

Comment:
1. Mmmh, it seems you're assuming some set `YARNRC`. Why? Isn't there a variable that `yarn` reads so we can make this `orchestra`-agnostic?
2. I guess the idea of `YARNRC` is that makes sure you don't fetch anything from the internet but it's fetched from the local mirror, is that right?
3. You're copying stuff from the source dir, but CMake does not about this. I think you should use `configure_file` in CMake and drop the script entirely in favor of just invoking `yarn install` from CMake.

Reply:
1. Rechecked, there is in fact a variable. Updated orchestra PR
2. Yes, basically it sets the correct path to the local cache and tells `yarn install` to use `--offline`
3. Done